internal/kafka/internal/services/kafkatlscertmgmt/secure_storage.go

```diff
@@ -194,5 +199,9 @@ func (storage *secureStorage) String() string {
 }
 
 func (storage *secureStorage) constructSecretName(key string) string {
-	return fmt.Sprintf("%s/%s", storage.secretPrefix, key)
+	return fmt.Sprintf("%s%s", storage.constructAWSSecretManagerKeysPrefix(), key)
```

Comment:
why no slash here? Is this already part of what `constructAWSSecretManagerKeysPrefix` returns?

Reply:
Yes. That's correct, it is part of that method call.